migration/migrating-4-4/migrating-openshift-4_1-to-4.adoc

```diff
@@ -1,4 +1,4 @@
-[id="migrating-openshift-4-to-4"]
+[id="migrating-openshift-4_1-to-4"]
```

Comment:
With changing file names for published files, you might need to set up redirects. I can't remember if this was done when the files got reorged for adding 4->4 initially. But if you haven't, should check with @vikram-redhat on this.

Reply:
OK, I'll ask about it.